satpy/etc/readers/mirs.yaml

```
@@ -5,6 +5,11 @@ reader:
   long_name: MiRS Level 2 Swath Product Reader (NetCDF4)
   reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
   sensors: [amsu, amsu-mhs, atms, ssmis, gmi]
+  data_files:
+     - url: "https://zenodo.org/record/4472664/files/limbcoef_atmsland_noaa20.txt"
```

```suggestion
     - url: "https://zenodo.org/record/4472664/files/limbcoef_atmsland_noaa20.txt"
       known_hash: "sha256:08a3b7c1594a963610dd864b7ecd12f0ab486412d35185c2371d924dd92c5779"
```

Generated by doing `openssl sha256 <file>` and copying the produced hash and preprending `sha256:`. Could you add these to all the files?